frappe/core/doctype/server_script/server_script.json

```diff
@@ -72,10 +73,17 @@
   {
    "fieldname": "section_break_8",
    "fieldtype": "Section Break"
+  },
+  {
+   "depends_on": "eval:doc.script_type==='Scheduler Event'",
+   "fieldname": "scheduler_event",
+   "fieldtype": "Select",
+   "label": "Scheduler Event",
+   "options": "All\nHourly\nHourly Long\nDaily\nDaily Long\nWeekly\nWeekly Long\nMonthly\nMonthly Long\nCron\nYearly\nAnnual"
```

Comment:
don't think we should have this inside server script. then we can have the same script job enabled to run multiple times just from scheduler doctype.

i think it is also redundant to have it here and in scheduler as well.

Reply:
i hope what i am saying is understandable:

creating server script document and setting scheduler event inside it will require you to create multiple duplicate server script documents if you want the same script to run at different times in a day

but removing this from here and handling it through scheduler will make sure you have only one server script mapped to multiple scheduler events